bloom-housing/bloom
# 1428/community type

# Pull Request Template

## Issue

Addresses #1428

## Description

Adds the community type section to listings management in partners, which lets you choose a reserved community type that will apply to an entire listing (we are starting with just Seniors 55+ and Seniors 62+).

This required removing `reservedType` from units (and thus a lil refactor to remove the reserved unit summaries) and using the reservedCommunityType which is on listings. The issue also indicated that we should be able to add custom descriptor text to our reserved type, so I added one backend field for that. There is already a descriptor on the reservedCommunityType object but in order for it to be unique to listings it seemed like it needed to be separate.

There were also a few frontend changes here to take the reserved indicators out of the summary tables and onto the Image Card.

## Type of change


## How Can This Be Tested/Reviewed?

Use the new Community Type section in listings management and preview listings. Additionally I added a reserved listing to our general seed data so if you re-seed you should see an already published reserved listing which will have all the new frontend pieces.

## Checklist:
